index.html

```diff
@@ -10,7 +10,50 @@
 </head>
 
 <body>
-    <h1>Hello World!</h1>
+    <!-- home page header -->
+    <main>
+        <header>
+            <!-- Mobile network icons -->
+            <div class="mobile-icons">
+                <p class="time">9:41</p> 
+                <img src="images/bar.png" alt="" height="15">
+            </div>
```

Comment:
- [ ] Kindly remove this top `mobile-icons` section, It's not a part of the design look 👇🏻 

- Your Design:

![18](https://user-images.githubusercontent.com/79058364/185663013-5f04688c-fd8d-469f-aa22-ed4f3b137a0b.png)




- Figma Design:

![image](https://user-images.githubusercontent.com/79058364/185663257-7be1859b-b9f3-4c86-933b-57402c78a5ad.png)



- Follow the original vision if it's pretty and works, it will be a wonderful website.. 🎨 🖌️